### ISSUE TITLE
Use PlantUML from MAVEN instead of JAR

### ISSUE THREAD

**Issue by costateixeira** [HL7/ig-template-base#224]
It's available here: 
https://repo1.maven.org/maven2/net/sourceforge/plantuml/plantuml-mit/1.2023.7/

**Comment by lmckenzi** [comment]
But the template process isn't tied into Maven...

**Comment by costateixeira** [comment]
no, but publisher is afaik... perhaps this issue belongs to publisher?

**Comment by lmckenzi** [comment]
Are you calling the publisher or are you calling plantUML from the templates?
The publisher already uses Maven

**Comment by lmckenzi** [comment]
@grahamegrieve will explore feasibility of including in publisher.
